src/Nancy/Request.cs

```diff
@@ -316,5 +318,13 @@ private void RewriteMethod()
 
             this.Method = providedOverride.Single().Item2;
         }
+
+        private string DebuggerDisplay
+        {
+            get
+            {
+                return String.Format("{0} {1}", this.Method, this.Url);

```

Comment:
On the response it makes sense to include the Content-Type? I know it's not on the RFC but makes sense from a debugging point of view